TOC.md

```diff
@@ -455,6 +455,7 @@
       + [Overview Page](/dashboard/dashboard-overview.md)
       + [Cluster Info Page](/dashboard/dashboard-cluster-info.md)
       + [Key Visualizer Page](/dashboard/dashboard-key-visualizer.md)
+      + [Metrics Relation Map](/dashboard/dashboard-metrics-relation.md)
```

Comment:
```suggestion
      + [Metrics Relation Graph](/dashboard/dashboard-metrics-relation.md)
```

Reply:
Please replace all `map` to `graph`.

Reply:
Done in https://github.com/pingcap/docs/pull/4015/commits/c5a9ea5969ff31df8479aae10d78bd179058b6d6.